src/Microsoft.Health.Fhir.Core/Extensions/DateTimeOffsetExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Fhir.Core.Extensions
+{
+    public static class DateTimeOffsetExtensions
+    {
+        private const string DateTimeOffsetFormat = "yyyy-MM-dd'T'HH:mm:ss.FFFFFFFK";
+
+        public static string ToInstantString(this DateTimeOffset offset)
```

Comment:
Might want to explain that this formats in UTC ISO 8601, and differs from standard format string "o" in that the fractional digits are omitted once zero.
(Although using "o" is ~5x faster because it is special-cased and they do not need to interpret the format string. But this is unlikely to be an issue.).